great_expectations/validator/validation_graph.py

```diff
@@ -111,7 +111,35 @@ def build_metric_dependency_graph(
                     runtime_configuration=runtime_configuration,
                 )
 
-    def resolve_validation_graph(  # noqa: C901 - complexity 16
+    def resolve_validation_graph(
+        self,
+        runtime_configuration: Optional[dict] = None,
+        min_graph_edges_pbar_enable: int = 0,
+        # Set to low number (e.g., 3) to suppress progress bar for small graphs.
+        show_progress_bars: bool = True,
+    ) -> Tuple[
+        Optional[Dict[Tuple[str, str, str], MetricValue]],
+        Dict[
+            Tuple[str, str, str],
+            Dict[str, Union[MetricConfiguration, Set[ExceptionInfo], int]],
+        ],
```

Comment:
This is a really really complex return - is there a way to make this a bit more legible?

Reply:
@cdkini I will be thinking about it -- this part will play in the BMS implementation, so might put it into a `dataclass` as well (in a separate pull request).  Thank you.